README.md

```diff
@@ -212,6 +215,22 @@ Response:
 }
 ```
 
+### Filtering Subtitles
+
+If you'd like to filter subtitles, one way to do so is by using regexing. asbplayer will match any sequence following regex pattern and remove the matches.

```

Comment:
Same comment as above about "filtering subtitles."
Also I would not consider "regexing" to be a commonly-used expression, even for more technical people.

> asbplayer will match any sequence following regex pattern and remove the matches.
- "a regex pattern" or "a specified regex pattern"
- asbplayer does not necessarily "remove the matches" even though that is a common use-case. Strictly speaking asbplayer replaces matched text with specified replacement text, which you have detailed below. So I at the very least I would say "can match ... and remove" instead of "will match ... and remove."

Reply:
OK, got it, making these changes now.
Should I mention that there is a difference between English () and Japanese（）?
something like
> Note: Japanese parentheses are usually different characters than the ones found on english keyboards: "()" versus "（）"

Or should I leave that to the user to discover since this is already kind of a niche thing?